### DISCUSSION TITLE
Node18 support?

### DISCUSSION THREAD

**Discussion by mprat** [requarks/wiki#5668] [Help / Questions] [answered]
Hi wikijs admins! I am using wikijs but am running into issues with Node 16 (specifically a memory leak in Node 16) that has been fixed in Node 18. I was wondering when in the roadmap there will be official support for Node 18?

**Comment by cdn0x12** [comment]
v3.0 will target and require node18.

**Comment by NGPixel** [comment, marked as answer]
Some dependencies are incompatible with Node 18 and would require many non-trivial changes to make it work. As such, Node 16 is the latest supported version for 2.x

Wiki.js 3.x is built with Node 18 in mind and will be the minimum supported version.
